libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -1098,6 +1101,12 @@ MAV_MISSION_RESULT AP_Mission::mavlink_int_to_mission_cmd(const mavlink_mission_
         cmd.p1 = packet.param1;                        // action 0=disable, 1=enable
         break;
 
+    case MAV_CMD_DO_SEND_SCRIPT_MESSAGE:
```

Comment:
If we are going to do this, do we want to just declare a couple of these? The reason I'm asking is that without an ID of some form, one of the floats is quickly going to be dedicated to being an ID for the script to use to make decisions, whereas if we have say 10 in the general enum reserved as SCRIPTING_MESSAGE_x they could all use the same entry here, and we can pass that ID to the script.